### PR TITLE
JBDS-4153 - correct Windows supplememtal runtime installation

### PR DESCRIPTION
**Pull request by pleacu**
Avoid backslash separator for WIndows and check for 'null' when calculating agg size.

@dgolovin - could you review/push?  thkx!!

### CODE REVIEW

**Comment by pleacu** [comment]
@dgolovin @alexeykazakov - sorry to be a pain - can this make it in today??

**Comment by alexeykazakov** [comment]
@pleacu master is frozen until tomorrow. Can it wait or it's super urgent for you?

File: installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java [hunk 1]
```diff
@@ -65,11 +64,12 @@ public void panelActivate() {
    private static long calculateAggregateSize(String sizes)
    {
       long totalSize = 0;
-      if (sizes != null && !sizes.isEmpty()) {
+
+      if (sizes != null && !sizes.trim().isEmpty() && !sizes.equals("null")) {
```
Comment (dgolovin) [inline installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java]: This feels wrong, did we miss null check somewhere while reading from json file?

Comment (pleacu) [inline installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java]: I added that as an extra precaution in the event that a size field was left out

File: installer/src/panels/com/izforge/izpack/installer/Unpacker.java [hunk 2]
```diff
@@ -866,7 +866,10 @@ void processRuntimes(String rtLocations, String installLocation, String installe
 	    	// Standard jar file extraction if an RT server exists.
 	    	if (runtimeEntry != null) {
 			handler.progress(i + 1, "Runtime " + runtimeEntry.getName());
-	    		String[] rtlComponents = rtl[i].split(java.io.File.separator);
+				String separator = java.io.File.separator;
```
Comment (dgolovin) [inline installer/src/panels/com/izforge/izpack/installer/Unpacker.java]: This should be just 
``` java
String[] rtlComponents = rtl[i].split("/");
```

File: installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java (hunk 1 above)
Comment (dgolovin) [inline installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java]: This is obviously problem in jaso file loading. This check is good for now as workaround.

**Comment by alexeykazakov** [comment]
Master is open. @dgolovin does it look good enough for you to merge it now?

**Pull request merged**